tests/test_behavior/test_audio_interface.py

```diff
@@ -0,0 +1,216 @@
+import os
+import shutil
+from copy import deepcopy
+from datetime import datetime
+from pathlib import Path
+from tempfile import mkdtemp
+
+import numpy as np
+from dateutil.tz import gettz
+from hdmf.testing import TestCase
+from numpy.testing import assert_array_equal
+from pynwb import NWBHDF5IO
+from scipy.io.wavfile import read, write
+
+from neuroconv import NWBConverter
+from neuroconv.datainterfaces.behavior.audio.audiointerface import (
+    AudioInterface,
+)
+from neuroconv.utils import FilePathType
+
+
+def create_audio_files(
+    test_dir: FilePathType,
+    num_audio_files: int,
+    sampling_rate: int,
+    num_frames: int,
+):
+    for audio_file_ind in range(num_audio_files):
+        write(
+            filename=Path(test_dir) / f"test_audio_file_{audio_file_ind}.wav",
+            rate=sampling_rate,
+            data=np.random.randn(num_frames, 2).astype(int),
```

Comment:
```suggestion
            dtype = "int32"
            np.random.randint(size=(12,5), low=np.iinfo(dtype).min, high=np.iinfo(dtype).max, dtype=dtype)
```

Reply:
Now here is where I'm curious - what `dtype` is in the test files themselves?

Reply:
The audio file from the Fee lab is in `int16`.